docs/lesiaturuta/PokemonProject/style.css

```diff
@@ -0,0 +1,762 @@
+  align-items: center;
+}
+
+.login__form-title {
+  margin-top: 10px;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 32px;
+  color: #221F1F;
+}
+
+.login__form-email,
+.login__form-password {
+  width: 100%;
+  height: 108px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+}
+
+.login__form-email-input,
+.login__form-password-input {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #A9A9A9;
+  border-radius: 6px;
+  padding-left: 20px;
+}
+
+.login__form-email-input::placeholder,
+.login__form-password-input::placeholder {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+  color: #A9A9A9;
+}
+
+.login__form-email-label,
+.login__form-password-label {
+  font-size: 22px;
+  font-weight: 600;
+  line-height: 26px;
+  color: #221F1F;
+}
+
+.login__form-checkbox {
+  width: 100%;
+  position: relative;
+}
+
+.block-select {
+  display: flex;
+  width: 69px;
+  height: 24px;
+  border-radius: 3px;
+  color: #FFF;
+  justify-content: center;
+  align-items: center;
+}
+
+.login__form-button {
+  width: 100%;
+  height: 70px;
+  padding: 27px 258px;
+  border-radius: 6px;
+  background: #EF4934;
+  font-size: 22px;
+  font-weight: 600;
+  line-height: 26px;
+  color: #FFF;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: none;
+}
+
+/*RESULT*/
+
+.result {
+  width: 100%;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.result__title {
+  margin-bottom: 30px;
+}
+
+.result__link {
+  text-decoration: none;
+}
+
+/*CALCULATE*/
+
+.calculate__title {
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 48px;
+}
+
+.field {
+  width: 100%;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+}
+
+.field__label {
+  font-size: 22px;
+  font-weight: 600;
+  line-height: 26px;
+  color: #221F1F;
+}
+
+.calculate__first-value,
+.calculate__operation,
+.calculate__second-value,
+.calculate__button {
+  width: 122px;
+  height: 36px;
+  border-radius: 6px;
+  border: 1px solid #221F1F;
+  text-align: center;
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 21px;
+}
+
+.calculate__result {
+  height: 36px;
+}
+
+.calculate__first-value,
+.calculate__operation,
+.calculate__second-value {
+  color: #221F1F;
+}
+
+.calculate__button {
+  background: #EF4934;
+  color: #FFF;
+  border: none;
+}
+
+.calculate__result {
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 36px;
+  color: #221F1F;
+}
+
+.calculate__show {
+  width: 100%;
+  margin-top: 80px;
+  display: flex;
+  font-size: 48px;
+  font-weight: 700;
+  line-height: 48px;
+  color: #EF4934;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.calculate__show-operation,
+.calculate__show-equals {
+  margin: 0 100px;
+}
+
+/*ERRORS*/
+
+.error-text {
+  position: absolute;
+  bottom: -20px;
+  left: 0;
+  color: #FA0A0A;
+}
+
+.border-error {
+  border: 4px solid #FA0A0A !important;
```

Comment:
These styles are working without `!important`. Please avoid unnecessary usage of this super-power.

Reply:
fixed